parser/schema_test.yml

```diff
@@ -136,6 +136,12 @@ object types:
                     Name: "argTwo"
                     Type: Int
                 Type: String
+  - name: must define one or more fields
+    input: |
+      type Hello {}
+    error:
+      message: "Unexpected {"
```

Comment:
With the proposed change this would read:
`expected at least one fields, found }`